blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,98 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is an `enterprise`_ feature.
+
+Enabling the JMX API
+--------------------
+
+.. highlight:: none
+
+To monitor CrateDB using the JMX API, you must to set the following system properties before you start CrateDB::
```

Comment:
All the dependencies should be very clear at this point.
* you need to run the enterprise editition
* you need to have stats enabled
* you need to have jmx settings set

it is fine to link to other sections for the details, but the user should know what he needs to do.